libdevcrypto/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set(EXECUTABLE devcrypto)
 
 file(GLOB HEADERS "*.h")
 add_library(${EXECUTABLE} ${SRC_LIST} ${HEADERS})
-add_definitions(-D_GLIBCXX_USE_CXX11_ABI=1)
+#add_definitions(-D_GLIBCXX_USE_CXX11_ABI=1)
```

Comment:
when cross compile with -mmic, use of symbol D_GLIBCXX_USE_CXX11_ABI results in many std::__cxx11::basic_string linker errors.

```
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:449:../libdevcrypto/libdevcrypto.so: undefined reference to `CryptoPP::RandomNumberGenerator::GenerateIntoBufferedTransformation(CryptoPP::BufferedTransformation&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long long)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:451:../libdevcrypto/libdevcrypto.so: undefined reference to `CryptoPP::BufferedTransformation::SetRetrievalChannel(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:452:../libdevcrypto/libdevcrypto.so: undefined reference to `dev::contentsString(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:453:../libdevcrypto/libdevcrypto.so: undefined reference to `CryptoPP::RandomPool::GenerateIntoBufferedTransformation(CryptoPP::BufferedTransformation&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long long)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:454:../libdevcrypto/libdevcrypto.so: undefined reference to `CryptoPP::BufferedTransformation::ChannelCreatePutSpace(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long&)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:455:../libdevcrypto/libdevcrypto.so: undefined reference to `dev::writeFile(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, dev::vector_ref<unsigned char const>, bool)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:456:../libdevcrypto/libdevcrypto.so: undefined reference to `vtable for CryptoPP::StringSinkTemplate<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:457:../libdevcrypto/libdevcrypto.so: undefined reference to `dev::fromHex(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, dev::WhenError)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:458:../libdevcrypto/libdevcrypto.so: undefined reference to `CryptoPP::BufferedTransformation::ChannelFlush(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, bool, int, bool)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:460:../libdevcrypto/libdevcrypto.so: undefined reference to `CryptoPP::BufferedTransformation::ChannelMessageSeriesEnd(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int, bool)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:461:../libdevcrypto/libdevcrypto.so: undefined reference to `CryptoPP::BufferedTransformation::ChannelPutModifiable2(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned char*, unsigned long, int, bool)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:462:../libdevcrypto/libdevcrypto.so: undefined reference to `dev::isHex(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:463:../libdevcrypto/libdevcrypto.so: undefined reference to `CryptoPP::BufferedTransformation::ChannelPut2(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned char const*, unsigned long, int, bool)'
/mnt/nfs/System/build_logs/libethereum-build_bot-20160429061730.txt:464:../libdevcrypto/libdevcrypto.so: undefined reference to `dev::fromUUID(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
[root@localhost libethereum-buildbot]# 

```